packages/components/src/components/buttons/TooltipButton/TooltipButton.stories.tsx

```diff
@@ -10,7 +13,81 @@ export default meta;
 export const TooltipButton: StoryObj<typeof TooltipButtonComponent> = {
     args: {
         children: 'Button',
+        variant: 'primary',
+        size: 'medium',
         isDisabled: true,
+        isLoading: false,
+        isFullWidth: false,
+        iconAlignment: 'left',
+        title: 'Button title',
         tooltipContent: 'Example tooltip',
+        ...getFramePropsStory(allowedButtonFrameProps).args,
+    },
+    argTypes: {
+        children: {
+            table: {
+                type: {
+                    summary: 'ReactNode',
```

Comment:
I'm just curious, why do you need to specify the type?

Reply:
I don't think I do need, storybook can inherit many of them but earlier this summer I was told by you to specify type of each arg in storybook anyway, to be more clear with types and not just rely on storybook itself.

Reply:
That's true, I was just thinking what does it mean `summary: 'ReactNode'` here

Reply:
The `summary` field within the `type` object provides a brief description of the type. In this case, it indicates that the `children` property is expected to be of type `ReactNode`.

It's the way how to provide more specific, more custom type in storybook, different than just simple string or number or array. I find it better to use this, better than just writing "string" here instead (as it is incorrectly in some other places and IMHO we should fix it). This way helps Storybook generate more accurate and informative documentation for the component.